{"version":3,"sources":["../../../../widgets/CartoMaintenanceWgt/case/CustomException.js"],"names":["define","ErrorEqualUrbanLotWithinBlock","message","name","Error"],"mappings":";;;;;;AAAA;AACAA,OAAO,EAAP,EAAW,YAAY;AAAA,QACbC,6BADa;AAAA;;AAEf,+CAAYC,OAAZ,EAAqB;AAAA;;AAAA,sKACXA,WAAW,sHADA;;AAEjB,kBAAKC,IAAL,GAAY,+BAAZ;AAFiB;AAGpB;;AALc;AAAA,MACyBC,KADzB;;AAQnB,WAAO;AACHH,uCAA+BA;AAD5B,KAAP;AAGH,CAXD","file":"CustomException.js","sourcesContent":["// CustomException.js\r\ndefine([], function () {\r\n    class ErrorEqualUrbanLotWithinBlock extends Error {\r\n        constructor(message) {\r\n            super(message || \"Se ha encontrado un lote urbano con la misma denominación dentro del mismo ámbito de unidad urbana y manzana urbana.\");\r\n            this.name = \"ErrorEqualUrbanLotWithinBlock\";\r\n        }\r\n    }\r\n\r\n    return {\r\n        ErrorEqualUrbanLotWithinBlock: ErrorEqualUrbanLotWithinBlock\r\n    };\r\n});"]}